std/algorithm/comparison.d

```diff
@@ -1565,3 +1569,367 @@ unittest
                 2, "two",
                 ));
 }
+
+/**
+Returns $(D true) if both ranges have the same length. This function is optimized
```

Comment:
Seems redundant to have a sentence that says `Returns` and a `Returns:` section that states what is already obvious (the return type is explicitly named). What about moving this sentence after `Params:` and turn it into the `Returns:` section?